lib/svgo/coa.js

```diff
@@ -112,6 +117,15 @@ async function action(args, opts, command) {
     }
   }
 
+  if (opts.eol != null) {
```

Comment:
Null check is not necessary anymore

Reply:
Ah, no. My bad. All good.

Reply:
if statements can be merged together

Reply:
Addressed in https://github.com/svg/svgo/pull/1546/commits/c413409d3e68e53c78679dd1118cc5adffa145a7.